.travis.yml

```diff
@@ -10,19 +9,18 @@ addons:
     packages:
       - g++-4.8
 env:
-  matrix:
-    - TRAVIS_NODE_VERSION="6"
-    - TRAVIS_NODE_VERSION="8"
+  jobs:
     - TRAVIS_NODE_VERSION="10"
-    - TRAVIS_NODE_VERSION="11"
+    - TRAVIS_NODE_VERSION="12"
+    - TRAVIS_NODE_VERSION="14"
 
 install:
   - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
   - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export CXX=g++-4.8; fi
   - $CXX --version
   - PATH="`npm bin`:`npm bin -g`:$PATH"
   # Install dependencies and build
-  - npm install
+  - NODE_GYP_FORCE_PYTHON=python2 npm install
```

Comment:
Do we really need legacy Python?!?  It died over a year ago and should now be considered an insecure platform.